lib/workers/branch/reuse.ts

```diff
@@ -60,6 +60,7 @@ export async function shouldReuseExistingBranch(
         logger.debug('Cannot rebase branch as it has been modified');
         return { reuseExistingBranch: true, isModified: true };
       }
+      logger.debug('Branch has no changes, but we will not use it');
```

Comment:
I don't like this message. It's written to sound "bad", but this is normal. The branch needs rebasing, and it is unmodified. 
```suggestion
```

Reply:
I just want to made a line in the log to be 146% sure which branch we were exiting this function. 
any suggestions are appreciated :D